distribution/lib/Standard/Table/0.0.0-dev/src/Fixed_Width/Fixed_Width_Format.enso

```diff
@@ -0,0 +1,202 @@
+from Standard.Base import all
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+import Standard.Base.Runtime.Ref.Ref
+import Standard.Base.System.File.Generic.Writable_File.Writable_File
+import Standard.Base.System.File_Format.File_Name_Pattern
+import Standard.Base.System.File_Format_Metadata.File_Format_Metadata
+import Standard.Base.System.Input_Stream.Input_Stream
+from Standard.Base.Errors.Common import Type_Error
+from Standard.Base.Metadata.Choice import Option
+
+import project.Data_Formatter.Data_Formatter
+import project.Delimited.Invalid_Fixed_Width_Rows.Invalid_Fixed_Width_Rows
+import project.Internal.Fixed_Width_Reader
+import project.Internal.Telemetry
+import project.Table.Table
+
+## Read fixed-width files into a Table.
+
+   A fixed-width format defines its columns as fixed column offsets and widths.
+   The columns are defined in a `Fixed_Width_Layout` object.
+type Fixed_Width_Format
+    ## Read fixed-width files into a Table.
+
+       Fixed-width files require have a `Fixed_Width_Layout` that specifies the
+       start and width of each column, along with an optional column name. This
+       layout can be defined using the `Fixed_Width_Column_Description` type, or
+       using an in-memory table.
```

Comment:
```suggestion
       using a table.
```
IMO this is going into too much detail, for users  in this context a table is just a table. 

I guess they are extremely unlikely to even consider using a DB table for this. And IF they do they should get a nice error from `Table.from DB_Table` conversion that will tell them to use `.read` first.